### PR TITLE
Change line endings in the json to see if that fixes the test on Windows

### LINKED ISSUE #1006: Tests on Windows fail with "Syntax error, malformed JSON"
- PHPMD version: 957fe4bacc91e6be8f0a4094547a81e762bfe65c
- PHP Version: 7.4.33 
- Installation type: composer
- Operating System / Distribution & Version: (Windows 10 / AppVeyor

### Current Behavior

```
There were 2 failures:
1) PHPMD\Renderer\SARIFRendererTest::testRendererCreatesExpectedNumberOfJsonElements
Failed asserting that '{\n    "version": "2.1.0",\n   ...  ]\n}\n' is valid JSON (Syntax error, malformed JSON).
C:\projects\phpmd\src\test\php\PHPMD\AbstractStaticTest.php:188
C:\projects\phpmd\src\test\php\PHPMD\Renderer\SARIFRendererTest.php:78
phpvfscomposer://C:\projects\phpmd\vendor\phpunit\phpunit\phpunit:51
2) PHPMD\Renderer\SARIFRendererTest::testRendererAddsProcessingErrorsToJsonReport
Failed asserting that '{\n    "version": "2.1.0",\n   ...  ]\n}\n' is valid JSON (Syntax error, malformed JSON).
C:\projects\phpmd\src\test\php\PHPMD\AbstractStaticTest.php:188
C:\projects\phpmd\src\test\php\PHPMD\Renderer\SARIFRendererTest.php:119
phpvfscomposer://C:\projects\phpmd\vendor\phpunit\phpunit\phpunit:51
FAILURES!
Tests: 714, Assertions: 943, Failures: 2.
Command exited with code 1
```

https://ci.appveyor.com/project/phpmd/phpmd/builds/47150788

Reproduced on my local Win 10.

### Expected Behavior

Tests should pass on Windows, too.

### Steps To Reproduce:

- Checkout with PhpStorm (likely irrelevant)
- Composer Installation through PhpStorm (likely irrelevant)
- ``composer test`` (likely irrelevant)

## Checks before submitting

Done

### PR DESCRIPTION
**Pull request by tvbeek**
Type: bugfix for the test
Issue: Resolves #1006 
Breaking change: no

I suspect it is a line ending issue so this is a try.

### CODE REVIEW

**Comment by kylekatarnls** [comment]
Good catch!

File: src/test/php/PHPMD/Renderer/SARIFRendererTest.php [hunk 1]
```diff
@@ -69,13 +70,19 @@ public function testRendererCreatesExpectedNumberOfJsonElements()
...
+        );
+
         $this->assertJsonEquals(
             $writer->getData(),
-            'renderer/sarif_renderer_expected.sarif',
-            function ($actual) {
-                $actual['runs'][0]['tool']['driver']['version'] = '@package_version@';
-                return $actual;
-            }
+            'renderer/sarif_renderer_expected.sarif'
```
Comment (kylekatarnls) [inline src/test/php/PHPMD/Renderer/SARIFRendererTest.php:85]: I guess this should be reverted to pass the test:
```suggestion
            'renderer/sarif_renderer_expected.sarif',
            function ($actual) {
                $actual['runs'][0]['tool']['driver']['version'] = '@package_version@';
                return $actual;
            }
```

File: src/test/php/PHPMD/Renderer/SARIFRendererTest.php [hunk 2]
```diff
@@ -112,11 +119,7 @@ public function testRendererAddsProcessingErrorsToJsonReport()
 
         $this->assertJsonEquals(
             $writer->getData(),
-            'renderer/sarif_renderer_processing_errors.sarif',
-            function ($actual) {
-                $actual['runs'][0]['tool']['driver']['version'] = '@package_version@';
-                return $actual;
-            }
+            'renderer/sarif_renderer_processing_errors.sarif'
```
Comment (kylekatarnls) [inline src/test/php/PHPMD/Renderer/SARIFRendererTest.php:122]:
```suggestion
            'renderer/sarif_renderer_processing_errors.sarif',
            function ($actual) {
                $actual['runs'][0]['tool']['driver']['version'] = '@package_version@';
                return $actual;
            }
```

**Review by AJenbo — DISMISSED** [review]

**Comment by tvbeek** [comment]
I suspect the main question is: do we want to keep the difficult to debug test on Windows?

**Comment by AJenbo** [comment]
no :D

Or more serious, WSL should allow Windows users to debug pretty easily I think?

**Comment by tvbeek** [comment]
> no :D
> 
> Or more serious, WSL should allow Windows users to debug pretty easily I think?

WSL on Windos != Windows. It is Linux (with some edge cases) And besides that we currently run on PHP 7.1

But I will take a look with getting it working on Windows.

**Review by AJenbo — APPROVED** [review]

**Comment by kylekatarnls** [comment]
Back to 🟢 🎉

**Pull request merged**